sdk/python/foundation-models/system/inference/image-text-embeddings/text-to-image-retreival.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Text-to-Image Retrieval using Online Endpoints and Indexes in Azure AI Search\n",
+    "\n",
+    "This sample shows how to perform vector search with a Azure AI Search Index and a deployed `embeddings` type model.\n",
+    "\n",
+    "### Task\n",
+    "Text-to-Image retrieval sends a text query to a search index and retrieves images closely related to the text query.\n",
```

Comment:
When talking about the task/problem, it would be better to state the inputs and outputs instead of describing what is happening in order to complete the task. Hastily put together suggestion: the task of t2i retrieval is to find from a collection of images those that are semantically related to a text query.